src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -578,25 +571,24 @@ void AlgorithmPaintToolBox::activateMagicWand()
 
 void AlgorithmPaintToolBox::behaveWithBodyVisibility()
```

Comment:
Maybe we should stop using `behaveWithBodyVisibility()` since it was meant to be used in the previous way of managing tbxes and it responds to the visibility of the body of the toolbox (so also called when maximisation, minimisation).
We could use now `showEvent` and `hideEvent` which fits perfectly our needs.
This can be done  in a later PR of course.